cn.navclub.dbfx.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id": "cn.navclub.dbfx",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "v-dbfx",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "flatpak_mysql",
+            "buildsystem": "cmake-ninja",
+            "cleanup": [
+                "lib",
+                "include"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "main",
+                    "url": "https://github.com/GZYangKui/flatpak-mysql.git",
+                    "commit": "a4d48c6323d4afa3ecf5b999ccab52e55d963881"
+                }
+            ]
+        },
+        {
+            "name": "dbfx",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "master",
+                    "url": "https://github.com/databasefx/v-dbfx.git",
```

Comment:
this repository has nothing to install.

Reply:
I'm also the first time to use flatpak to distribute applications. I don't know much about the distribution rules of private applications. Can you distribute relevant documents or examples to private applications?

Reply:
You need to store a binary somewhere (e.g. S3) and then "repackage" it here. You can take a look at https://github.com/flathub/com.discordapp.Discord for an example.